Formula/r/reminiscence.rb

```diff
@@ -1,11 +1,14 @@
 class Reminiscence < Formula
   desc "Flashback engine reimplementation"
   homepage "http://cyxdown.free.fr/reminiscence/"
-  # A mirror is used as the primary URL because the official one rate limits
-  # too heavily that CI almost always fails.
-  url "https://pkg.freebsd.org/ports-distfiles/REminiscence-0.5.1.tar.bz2"
-  mirror "http://cyxdown.free.fr/reminiscence/REminiscence-0.5.1.tar.bz2"
-  sha256 "6b02b8568a75af5fbad3b123d2efe033614091d83f128bc7f3b8b533db6e4b29"
+  url "https://github.com/cyxx/REminiscence/archive/refs/tags/0.5.2.tar.gz"
+  sha256 "e7ccfe348024dd7a0d893bac1d0b3efd04bf94f7cf0dd5335b23d154c826af96"
+
+  # The official URL is used only as a mirror because it rate limits too
+  # heavily that CI almost always fails.
+  mirror "http://cyxdown.free.fr/reminiscence/REminiscence-0.5.2.tar.bz2" do
+    sha256 "86874e1163451ae499f470a216d6c1f92097f769b968bf289a3e343eb7a5a3cf"
+  end
```

Comment:
I'm not 100% sure whether this block syntax for `mirror` would even work. I checked in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook) but didn't see `mirror` info. I also grepped the repo to see if other formulae had anything like this but didn't see anything.

Reply:
if the sha is different, then the artifact is different, so it is not technically a mirror.

Reply:
True. For my own education, is there any supported way to specify alternate URLs then?

Reply:
No, since there's not much value in downloading two different sources for the same formula.